### ISSUE TITLE
macOS: Add "settings" button to the toolbar of a virtual machine window

### ISSUE THREAD

**Issue by diamondsw** [utmapp/UTM#3481]
As a shortcut to adjust the configuration of a currently running VM, add a button to the toolbar that opens the settings window directly, with the understanding that most if not all changes won't take effect until the guest machine is rebooted.

**Comment by osy** [comment]
I think that would be confusing because none of the changes will take effect.

**Comment by diamondsw** [comment]
It's still useful to refer to how a machine is currently configured (for instance, selecting drivers and such during setup of an OS), or if you're in a reboot loop because something needs changing (as I found with WinXP running out of memory). Attached is an example of what Parallels does if you open the configure dialog while the VM is running.
<img width="758" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/49737/149438777-14b50af9-1671-453f-8d5d-584ef49585a2.png">

**Comment by conath** [comment]
+1 would like to have this, as well. Either a reduced number of settings are available or the ones that can’t be changed could be grayed out.

**Comment by osy** [comment]
Right now it's technically difficult as we use SwiftUI for the main window and AppKit for the VM windows.

**Comment by diamondsw** [comment]
Fair enough - a classic case of "looks simple on the surface, but not so much in implementation". Thanks for the consideration!
